wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -444,10 +390,9 @@ $errorExample
 |]
 
 data DescriptionEnvironment = DescriptionEnvironment {
```

Comment:
Record fields do not follow conventions here which make them prone to clash on import (..).

```haskell
data DescriptionEnvironment = DescriptionEnvironment {
    deErrorExample :: !Text
  , deDefaultPerPage :: !Text
  ...
}
```